### DISCUSSION TITLE
Table column title, sorter and filter alignment and justifying according to demand

### DISCUSSION THREAD

**Discussion by rustam-as** [ant-design/ant-design#43736] [General]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/fsp2sn?file=/demo.js)

### Steps to reproduce

I want to modify title, sorter and filter alignment on my column title area.

### What is expected?

My goal is to justify title, sorter and filter very close and center of the column

### What is actually happening?

I cant effect it

| Environment | Info |
| --- | --- |
| antd | 5.7.2 |
| React | react@18.2.0 |
| System | Windows 11 pro 64bit (10.0 Build 22000) |
| Browser | Google Chrome Version 110.0.5481.77 (Official Build) (64-bit) |

**Comment by linxianxi** [comment]
like this? 
<img width="256" alt="image" src="https://github.com/ant-design/ant-design/assets/47104575/f35762b8-18f5-4a50-8c63-738523a8ccf1">

https://codesandbox.io/s/filter-and-sorter-antd-5-7-2-forked-d3z7vh?file=/demo.js

**Reply by rustam-as** [reply]
yes exactly
